src/applications/caregivers/utils/constants.js

```diff
@@ -23,7 +23,11 @@ export const ADDRESS_REGEX = {
 export const ALLOWED_FILE_TYPES = ['pdf', 'jpg', 'jpeg', 'png'];
 
 export const API_ENDPOINTS = {
+  csrfCheck: '/maintenance_windows',
+  downloadPdf: '/caregivers_assistance_claims/download_pdf',
   facilities: '/caregivers_assistance_claims/facilities',
+  fileUpload: '/v0/form1010cg/attachments',
+  submission: '/v0/caregivers_assistance_claims',
```

Comment:
These endpoints that contain `v0` are utilized in the form config and file upload component, where there isn't the same check as in the `apiRequest` to auto-append the version. We need to explicitly declare their version for use.